openedx/core/lib/xblock_builtin/xblock_discussion/static/js/discussion_classes.js

```diff
@@ -0,0 +1,22 @@
+// Add RequireJS definitions for each discussion class
```

Comment:
@andy-armstrong I tested without this shim, and things worked fine. Removed it in https://github.com/edx/edx-platform/pull/13483/commits/724a839f77255346e911b8de2468d8ec5f4c68ea.